creator-node/src/services/stateMachineManager/stateMachineConstants.js

```diff
@@ -62,7 +67,9 @@ module.exports = {
     // Name of queue that only processes jobs to fetch the cNodeEndpoint->spId mapping,
     C_NODE_ENDPOINT_TO_SP_ID_MAP: 'c-node-to-endpoint-sp-id-map-queue',
     // Name of StateReconciliationQueue
-    STATE_RECONCILIATION: 'state-reconciliation-queue'
+    STATE_RECONCILIATION: 'state-reconciliation-queue',
+    // Name of ManualSyncQueue
+    MANUAL_SYNC: 'manual-sync-queue'
```

Comment:
since redis doesn't wipe state on service restart, we should make sure no previous state exists if we're gonna re-use name

Reply:
obliterating state is a good call for the reason you mentioned. I talked about the name and using reconciliation queue vs a new queue with @jonaylor89 and we thought it would be best to use a separate queue from recurring syncs because we want the manual-sync-queue to be as fast as possible to minimize time spent waiting on the client

also, we were careful to double-check that this PR removes the manual-sync-queue from snapback and instead sets this new queue as serviceRegistry's `this.manualSyncQueue`